integration_test/mysql/test_helper.exs

```diff
@@ -1,12 +1,30 @@
 Logger.configure(level: :info)
 
-# :modify_column is supported on MySQL 5.7
-# windows.exs is supported on MySQL 8.0
-# but that is not yet supported in travis.
-ExUnit.start exclude: [:array_type, :read_after_writes, :returning, :modify_column,
-                       :strict_savepoint, :create_index_if_not_exists, :aggregate_filters,
-                       :transaction_isolation, :rename_column, :with_conflict_target,
-                       :map_boolean_in_expression]
+defmodule TestHelper do
+  def mysql_8?() do
+    case System.get_env("MYSQLVERSION") do
```

Comment:
I don't want to depend on the system environment because then every user running the suite would have to set it.

We can actually move the `ExUnit.start` to the end of the file (and do the same for PG) and use the PoolRepo (or TestRepo) to ask the database itself its version.